docs/assets/js/admin/decap-cms.js

```diff
@@ -1,10 +1,12 @@
-import CMS from 'netlify-cms';
+import CMS from 'decap-cms-app';
 import { StringTrimmedControl } from './widgets/StringTrimmed.js';
 import { StringWithInstructionsControl } from './widgets/StringWithInstructions.js';
 import genericPreviewTemplate from './widgets/genericPreviewTemplate.js';
 import navigationPreviewTemplate from './widgets/navigationPreviewTemplate.js';
 import pagePreviewTemplate from './widgets/pagePreviewTemplate.js';
 
+CMS.init();
```

Comment:
It's weird that we need to initialize the CMS when the [docs](https://decapcms.org/docs/manual-initialization/) state that it's only necessary when the `CMS_MANUAL_INIT` flag is enabled. 🤷

Reply:
That is weird! But if I remove it the CMS screen is a blank screen 🤷